requirements/base.txt

```diff
@@ -1,14 +1,14 @@
 # the default package dependencies
-
+certifi>=14.05.14
 hydra-core>=1.0.0
+lightly_utils~=0.0.0
 numpy>=1.18.1
-pytorch_lightning>=1.0.4,<1.7  # TODO: Hotfix, remove upper bound
+python_dateutil>=2.5.3
+pytorch_lightning>=1.5.0
 requests>=2.23.0
-torchvision
+setuptools>=21.0.0
+six>=1.10
+torch>=1.6.0
```

Comment:
This would be a new dependency, no?

Reply:
It is not, we installed it as a second order dependency of `torchvision`.